src/components/Leaderboard/Leaderboard.jsx

```diff
@@ -0,0 +1,70 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import ReactPaginate from 'react-paginate';
+import './leaderboard.css';
+const Leaderboard = () => {
+  // eslint-disable-next-line no-unused-vars
+  const [allPlayers, setAllPlayers] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [currentPage, setCurrentPage] = useState(0);
+  const getPlayers = async () => {
+    setLoading(true);
+    // Grabbing players from the backend, later we could make this a little different
+    //--- Maybe we wouldn't want to get players who have 0 wins, or you have to gave a certain number
+    //--- of wins to be on the leaderboard?
+    const { data } = await axios.get('/api/player/allPlayers');
+    console.log(data);
+    // Sort in place based on wins, then set it to local state, although this can later be redux state
+    data.sort((a, b) => b.wins - a.wins);
+    setAllPlayers(data);
+    setLoading(false);
+  };
+  // PAGINATION
+  // Hardcoded # of books per page, if in the future we wanted the user to be able to change the # of items per page
+  // we would then could have state determine this.
+  const PER_PAGE = 5;
+  const offset = currentPage * PER_PAGE;
+  const currentPageData = allPlayers.filter((player) => player.wins > 3).slice(offset, offset + PER_PAGE);
```

Comment:
Can you explain a little more about this .filter callback?
Are you deciding here that in order to get onto the leaderboard you must have atleast 3 wins?
Because right now I noticed that players with 0 wins (newly created players) do not appear on the leaderboard, but they are in the `data` returned from the GET allPlayers.

If you are deciding on 3 for the cutoff, we should say that somewhere, so someone with 2 wins and trying to find themselves isn't confused